_scicomputing/hutchgo_guest_collection.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Creating Guest Collections in HutchGO
+last_modified_at: 2024-07-01
+primary_reviewers: atombaby
+---
+
+Guest collections are used in HutchGO to share data with people outside your immediate group.  You can share data with Hutch staff or external collaborators.
+
+To create a collection your lab must already be configured to use HutchGO, including setting up the base mapped collection- see documentation [here](scicomputing/hutchgo_overview) on setting that up.
```

Comment:
broken link

Reply:
Whoops- leading slash missing (fixed in 211f797).